data/json/recipes/recipe_food.json

```diff
@@ -4228,17 +4229,24 @@
 },{
   "type" : "recipe",
   "result": "syrup",
+  "id_suffix": "with_instrument",
```

Comment:
Currently no

I tried implementing it once, but gave up. I'll implement project crafting, then maybe it will look easier.